### PR TITLE
Fix on new doc

### PR DESCRIPTION
**Pull request by bidoubiwa**
```
I really LOVE the new structure!! (The new Home page, the separated part between Guide/Api Reference/Tutorial...)
Some points:

The search bar in the middle of the navbar chocks me a lot. Is it possible to put it at the right?
At the end of the QuickStart, the user is not redirected to another part of our documentation. Maybe the Guide part is a good idea...?
In Guide -> Getting started -> Download and launch :
1) there is a dead link on option available and the tests didn't see it.
2) I think it could be better to put all the possibilities of download like in Advanced Guide -> Installation (but we should find a way to link both tables)
```
@curquiza 
The searchbar position will be changed when I change the searchbar.
The user is now redirected to the getting started which in turn redirects to all parts of the documentation.
No dead links have been found in the getting started.
I have put all the possibilities of download in the getting started.

### CODE REVIEW

File: guides/README.md [hunk 1]
```diff
@@ -269,6 +342,20 @@ Meilisearch **response** :
 ```
 
 
+### Afterword
+
+
+In MeiliSearch we have three concepts on which we build our search engine. If you did not go to those pages, they give an essential insight.
```
Comment (curquiza) [inline guides/README.md]: I don't understand what you wanted to mean with the "If you did not go to those pages". Is it more "If you haven't read these pages yet, ..." ?

File: guides/README.md [hunk 2]
```diff
@@ -1,43 +1,135 @@
...
+### Download and launch
 
 First of all, you must have access to a running instance of MeiliSearch.
 
-### Download and launch
 There are [several download possibilities](/guides/advanced_guides/binary.md#download-and-launch).
 
-Once it is downloaded, we need to start our instance of MeiliSearch. The port, as well as the master key, can be defined at launch. There are other [options available](/guides/advanced_guides/binary.md#usage).
+:::: tabs
+
```
Comment (curquiza) [inline guides/README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -4,14 +4,17 @@
...
+
 Efficient search engines are often only accessible to companies with the financial means and resources necessary to develop a search solution adapted to their needs. The majority of other companies that do not have the means or do not realize that the lack of relevance of a search greatly impacts the pleasure of navigation on their application,
 end up with poor solutions that are more frustrating than effective, for both the developer and the user.
 
 That's why we created MeiliSearch, an open-source solution accessible to everyone, meeting the vast majority of needs, even specific ones. Installable very easily with little or no configuration required but with a high capacity for customization.
 
 Our solution is **instant**; it **accepts typos**; it understands **filters**, **custom rankings**, and a lot of other [features](/getting_started/features.md).
 
-<linkButton method="GET" text="🚀  QUICKSTART" url="/tutorials"/>
+
```
Comment (curquiza) [inline README.md]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -4,14 +4,17 @@
 
 MeiliSearch is a **RESTfull search API** that is the **ready-to-go solution** for everyone wanting a **powerful, fast, and relevant search experience** for their end-users ⚡️🔎
 
+<linkButton method="GET" text="🚀  GETTING STARTED" url="/guides/#getting-started"/>
+
+
```
Comment (curquiza) [inline README.md]:
```suggestion
```

File: guides/README.md [hunk 5]
```diff
@@ -269,6 +342,20 @@ Meilisearch **response** :
 ```
 
 
+### Afterword
+
```
Comment (curquiza) [inline guides/README.md]:
```suggestion
```

File: guides/advanced_guides/binary.md [hunk 6]
```diff
@@ -104,13 +104,8 @@ $ ./target/release/meilisearch
...
 
 
-## Flags
-[Flags](/guides/advanced_guides/binary.md#environment-variables) can be added on launch.
 
-```bash
-$ ./meilisearch --db-path ./meilifiles --http-addr 127.0.0.1:7700
-Server is listening on: http://127.0.0.1:7700
-```
+
```
Comment (curquiza) [inline guides/advanced_guides/binary.md]:
```suggestion
```

File: guides/README.md [hunk 7]
```diff
@@ -1,43 +1,135 @@
...
 There are [several download possibilities](/guides/advanced_guides/binary.md#download-and-launch).
 
-Once it is downloaded, we need to start our instance of MeiliSearch. The port, as well as the master key, can be defined at launch. There are other [options available](/guides/advanced_guides/binary.md#usage).
+:::: tabs
+
+::: tab cURL
+Download the **latest stable release** of MeiliSearch with **curl**.
 
-One way of installing MeiliSearch is curl :
+Launch the MeiliSearch package to start MeiliSearch's server.
```
Comment (curquiza) [inline guides/README.md]: Isn't it 'binary'?

Comment (bidoubiwa) [reply guides/README.md]: In this context is it from the APT installation, which is a package no ?

File: guides/README.md (hunk 1 above)
Comment (bidoubiwa) [reply guides/README.md]: Yes great suggestion

**Comment by curquiza** [comment]
You were fast!! :)
> No dead links have been found in the getting started.
Yes, I found no dead links in the previous merge either, it was fixed before the merge obviously.
>  The user is now redirected to the getting started which in turn redirects to all parts of the documentation.
I think it's a good idea :)

File: guides/README.md (hunk 7 above)
Comment (curquiza) [reply guides/README.md]: I mean, we don't 'launch' a package. We 'download' a package, and 'launch' a binary.

Comment (curquiza) [reply guides/README.md]: The binary starts the server.

Comment (curquiza) [reply guides/README.md]: But if everybody agrees with you, go for 'package' :)

**Pull request merged**